src/dfmpreview/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
```

Comment:
and for dfm_test.exe, should check in source code so we can review?

---
In reply to: [73278778](https://github.com/dotnet/docfx/pull/558#discussion_r73278778) [](ancestors = 73278778)